core/src/main/java/de/danoeh/antennapod/core/util/ShareUtils.java

```diff
@@ -59,17 +51,15 @@ public static boolean hasLinkToShare(FeedItem item) {
         return FeedItemUtil.getLinkWithFallback(item) != null;
     }
 
-    public static void shareFeedItemLink(Context context, FeedItem item, boolean withPosition) {
-        String text = getItemShareText(item) + " " + FeedItemUtil.getLinkWithFallback(item);
-        if (withPosition) {
-            int pos = item.getMedia().getPosition();
-            text += " [" + Converter.getDurationStringLong(pos) + "]";
+    public static void shareFeedItemLinkWithDownloadLink(Context context, FeedItem item, boolean withPosition) {
+        String text = getItemShareText(item) + " ";
+        if (hasLinkToShare(item)) {
+            text +=  "\n\n" + R.string.share_dialog_episode_website_label + ": ";
```

Comment:
`context.getResources.getString()` loads the string. Like this it only displays the ID of the string.